scripts/sites.js

```diff
@@ -90,6 +90,6 @@ const sites = [
   { url: 'https://amorris.ca', title: 'amorris', author: 'amorris', type: 'blog', feed: 'https://feed.amorris.ca/hallway.txt' },
   { url: 'http://www.miha-co.ca', title: 'miha-co', type: 'Portfolio' },
   { url: 'https://notes.stuartpb.com/', title: 'notes.stuartpb.com', type: 'wiki', author: 'stuartpb', contact: 's@stuartpb.com', feed: 'https://twtxt.stuartpb.com/xxiivv.txt' }
-  { url: 'http://xxiii.co' },
+  { url: 'http://xxiii.co' title: 'xxiii', type: 'portfolio', author: 'serocell', contact: 'psignal@s900.net', rss: 'http://serocell.com/feeds/serocell.xml', feed: 'http://xxiii.co/twtxt.txt' },
```

Comment:
Missing comma, please test before making PRs.

Reply:
is this why the webring is down?

Reply:
yeah I shouldn't have merged @serocell's PR, I'll revert :)

Reply:
Fixed, sorry about that :)